native/swift/Sources/wordpress-api/Extensions.swift

```diff
@@ -32,3 +32,23 @@ extension MiddlewarePipeline {
         self.init(middlewares: middlewares)
     }
 }
+
+public extension CommentStatus {
+    init(status: String) {
+        self = commentStatusFromString(value: status)
+    }
+
+    func status() -> String {
```

Comment:
Should this be a method or a property with a `get`?

I kinda think it should be the latter?

Reply:
Sounds good. Updated in https://github.com/Automattic/wordpress-rs/pull/734/commits/818599727353f51afe51a6f2824b50a002c90ec5